pandas/core/indexes/extension.py

```diff
@@ -164,6 +164,11 @@ class ExtensionIndex(Index):
 
     _data: ExtensionArray
 
+    def repeat(self, repeats, axis=None):
```

Comment:
```suggestion
    def repeat(self, repeats: Union[int, Sequence[int]], axis: Axis=None):
```

might be nice to add while moving